package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "yummy-squeegee",
+  "version": "0.0.1",
+  "description": "We're on some next shhhh",
+  "engines": {
+    "node": ">=5.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.0",
+    "immutable": "^3.7.5",
+    "jquery": "^2.1.4",
```

Comment:
HAHA. I should probably remove that. I'm not using it. I honestly don't know where it came from.